schemas/common/descriptors/relationshipdescriptor.schema.json

```diff
@@ -10,7 +10,7 @@
   "title": "Relationship Descriptor",
   "meta:extensible": true,
   "meta:extends": [
-    "https://ns.adobe.com/xdm/common/desciptors/schemadescriptor#/definitions/descriptor"
+    "https://ns.adobe.com/xdm/common/descriptors/schemadescriptor#/definitions/descriptor"
```

Comment:
This is being considered as remove action in allOf, because of which build is failing with incompatibility changes.